dsl/src/main/scala/com.crobox.clickhouse/dsl/misc/LogicalOperatorImprovements.scala

```diff
@@ -4,64 +4,64 @@ import com.crobox.clickhouse.dsl._
 
 object LogicalOperatorImprovements {
 
-  implicit class TableColumnImpr(left: TableColumn[Boolean]) {
+  implicit class TableColumnImpr(left: ExpressionColumn[Boolean]) {
```

Comment:
For consistency make sure the class is also `ExpressionColumnImpr`; 
But why is this made more strict? e.g. what's wrong with keeping it a `TableColumn`?

Reply:
Reverted to old situation